transcribed/page0320.txt

```diff
@@ -37,19 +37,19 @@
 037     100 023         JRST POPJ1
 038     001 048 CLKONJ: CONO PI,CLKON
 039     001 016         POPJ P,
-040
+040    
 041             ATRNDL:         ;CALL 7,
 042     117 028         MOVE A,SIOMTU
 043     001 007         MOVE B,(C)
 044     118 044         CAME B,UNAME(U)
 045     001 016         POPJ P,
 046             TRND:   IRPC N,,12345678
-047     001 007         MOVE B,TRAN1N(A)
+047     001 007         MOVE B,TRANIN(A)
```

Comment:
i belive this is TRAN!N, also in line 35

Reply:
Ah, tricky one!  Yes, those two are ! which is valid inside an IRP (or macro).